### DISCUSSION TITLE
How to parse content to wstring?

### DISCUSSION THREAD

**Discussion by magebot1** [nlohmann/json#2698] [Q&A] [answered]
```c++
		std::wstring content = Script::Url("","GET","");

		nlohmann::json jf = nlohmann::json::parse(content);

		std::wstring v = jf["key1"].get<std::wstring>();
		std::wstring v2 = jf["key2"].get<std::wstring>();
```
The code above only works with `std::string`, it's not possible to parse it to a `wstring`?

**Comment by nlohmann** [comment, marked as answer]
The library stores string internally as (UTF-8 encoded) `std::string`. If you want a `std::wstring`, you need to provide the conversion yourself.
